packages/net/http/http_notify.pony

```diff
@@ -45,6 +45,11 @@ interface tag HTTPSession
     submission of the HTTP message is complete.
     """
 
+  be dispose()
```

Comment:
Ya, it doesn't follow naming conventions for standard library.

But that could be in another PR:

https://github.com/ponylang/ponyc/blob/master/CONTRIBUTING.md#standard-library-file-naming